### ISSUE TITLE
Proposed Steps for Evolution Manuscript Submission

### ISSUE THREAD

**Issue by rando2** [greenelab/covid19-review#1084]
- Update with omicron
- Add estimates of general evolutionary parameters
- Add a bit of zoonotic context (which species get it, where it might have come from)

Aim for 5,000 words 

Anything else?

**Comment by agitter** [comment]
There are a few studies using deep mutational scanning to understand the impact of potential future viral evolution.  That is an interesting tangential topic, not an essential one.

**Comment by RLordan** [comment]
Can we somehow integrate the lovely pangolin/pando data on emerging variants, if we haven't already - I know I make that sound like a trivial thing :D 

https://cov-lineages.org/

**Comment by dziakj1** [comment]
This might be worth citing in the discussion somewhere:

Challenges of evidence synthesis during the 2020 COVID pandemic: a scoping review.
CITATION: J Clin Epidemiol. 2022 Feb;142:10-18. doi: 10.1016/j.jclinepi.2021.10.017. Epub 2021 Oct 27.
Khalil H(1), Tamara L(2), Rada G(3), Akl EA(4).
ABSTRACT
AIM: The objectives of this scoping review are to identify the challenges to conducting evidence synthesis during the COVID-19 pandemic and to propose some recommendations addressing the identified gaps. METHODS: A scoping review methodology was followed to map the literature published on the challenges and solutions of conducting evidence synthesis using the Joanna Briggs Methodology of performing scoping review. We searched several databases from the start of the Pandemic in December 2019 until 10th June 2021. RESULTS: A total of 28 publications was included in the review. The challenges cited in the included studies have been categorised into four distinct but interconnected themes including: upstream, Evidence synthesis, downstream and contextual challenges. These challenges have been further refined into issues with primary studies, databases, team capacity, process, resources, and context. Several proposals to improve the above challenges included: transparency in primary studies registration and reporting, establishment of online platforms that enables collaboration, data sharing and searching, the use of computable evidence and coordination of efforts at an international level. CONCLUSION: This review has highlighted the importance of including artificial intelligence, a framework for international collaboration and a sustained funding model to address many of the shortcomings and ensure we are ready for similar challenges in the future.

**Comment by rando2** [comment]
Hi authors (@cbrueffer @RLordan @Sergey-Knyazev @marouenbg @yusha-sun), just a heads up that I am trying to finish opening all the PRs we'll need by next week! If you are planning to add any more evolution text, please let me know. Otherwise, keep an eye out for an email for manuscript approval in the next week or so!

**Comment by RLordan** [comment]
I just see you suggested paths to submission, a brief mention of wastewater variant surveillance as something novel would be cool. I can get literature on that. I am a little bit tied up until after Tuesday for writing, but can help then.

**Comment by Sergey-Knyazev** [comment]
Sure, I can help with that. Thank you!

**Comment by rando2** [comment]
@RLordan and @Sergey-Knyazev that sounds fantastic! It has become increasingly important, so definitely good to discuss! So we have my 3-4 PRs and that one at a minimum that we'll wait for!

@RLordan I looked at Pangolin/Pando and I think I could scrape some data from there! What is your dream data to include and I'll see if I can do it? haha

**Comment by RLordan** [comment]
> @RLordan and @Sergey-Knyazev that sounds fantastic! It has become increasingly important, so definitely good to discuss! So we have my 3-4 PRs and the wastewater one at a minimum that we'll wait for!
> 
> @RLordan I looked at Pangolin/Pando and I think I could scrape some data from there! What is your dream data to include and I'll see if I can do it? haha

Yes, leave me think about this tonight, I will look through to see what makes most sense... there is a lot to choose from 😂

**Comment by RLordan** [comment]
> @RLordan and @Sergey-Knyazev that sounds fantastic! It has become increasingly important, so definitely good to discuss! So we have my 3-4 PRs and the wastewater one at a minimum that we'll wait for!
> 
> @RLordan I looked at Pangolin/Pando and I think I could scrape some data from there! What is your dream data to include and I'll see if I can do it? haha

Hey @rando2, So I think a general phylogenetic tree with the main variants would be good. The other thing that might be cool is something like figure 7 here for the most prevalent variants https://cov-lineages.org/global_report_B.1.617.2.html

**Comment by RLordan** [comment]
> I wonder whether Eric Topol would be willing to let you adapt one or more of his visualizations here:
> https://erictopol.substack.com/p/the-ba5-story

Yes, these are good visuals! I saw a few on his twitter.
